# Type: refactor
# Subtype: R10
# English Summary: Delete variable result
# Chinese Summary: 删除变量result
# Entry Number: 21

# === Git Diff Content ===

diff --git a/frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/Utils.java b/frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/Utils.java
index 29db578..7fcd794 100644
--- a/frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/Utils.java
+++ b/frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/Utils.java
@@ -47,12 +47,10 @@ final class Utils {
 
         public void run(){
             BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream));
-            String result = "";
             String line;
             try {
                 while((line = reader.readLine()) != null) {
-                    result += line + "
";
-                    log.info(result);
+                    log.info(line);
                 }
             } catch (IOException e) {
 
